certification/internal/bundle/bundle.go

```diff
@@ -211,3 +213,70 @@ func GetSupportedInstallModes(ctx context.Context, csvReader io.Reader) (map[str
 	}
 	return installedModes, nil
 }
+
+func ExtractImagesFromBundle(ctx context.Context, csvReader io.Reader) ([]string, error) {
```

Comment:
Wasn't this method removed previously?

Reply:
hmm weird it didnt go away with rebase will remove..

Reply:
I might be wrong, But I thought it was removed.

Reply:
you are right its not in main 👍